commune_app/all_models/users.py

```diff
@@ -38,10 +38,12 @@ def save(self, *args, **kwargs):
         return super().save(*args, **kwargs)
 
     def join_commune(self, commune_id):
+        commune = Commune.objects.filter(id=commune_id).first()
         if (self.commune_id is not None and self.commune_id != commune_id):
-            raise Exception("already partner in another commune")
+            raise Exception("already partner in the commune: " + self.commune_id.name)
```

Comment:
I think it should be like:
```suggestion
            raise Exception("already partner in the commune: " + self.name)
```

Reply:
self is user, not commune
So I'm pretty sure it should stay that way